host-spec/ad-netmessages.tm

```diff
@@ -1,400 +0,0 @@
-<TeXmacs|1.99.12>
-
-<project|host-spec.tm>
-
-<style|book>
-
-<\body>
-  <appendix|Network Messages><label|sect-network-messages>
-
-  In this section, we will specify various types of messages which the
-  Polkadot Host receives from the network. Furthermore, we also explain the
-  appropriate responses to those messages.
-
-  <\definition>
-    A <strong|network message> is a byte array, <strong|<math|M>> of length
-    <math|<around*|\<\|\|\>|M|\<\|\|\>>> such that:
-
-    \;
-
-    <\equation*>
-      <tabular*|<tformat|<table|<row|<cell|M<rsub|1>>|<cell|Message Type
-      Indicator>>|<row|<cell|M<rsub|2>\<ldots\>M<rsub|<around*|\<\|\|\>|M|\<\|\|\>>>>|<cell|Enc<rsub|SC><around*|(|MessageBody|)>>>>>>
-    </equation*>
-
-    \;
-  </definition>
-
-  The body of each message consists of different components based on its
-  type. The different possible message types are listed below in Table
-  <reference|tabl-message-types>. We describe the sub-components of each
-  message type individually in Section <reference|sect-message-detail>.
-
-  <big-table|<tabular*|<tformat|<cwith|2|-1|2|2|cell-halign|l>|<cwith|1|-1|1|-1|cell-tborder|0ln>|<cwith|1|-1|1|-1|cell-bborder|0ln>|<cwith|1|-1|1|-1|cell-lborder|1ln>|<cwith|1|-1|1|-1|cell-rborder|1ln>|<cwith|18|18|1|-1|cell-bborder|1ln>|<cwith|1|-1|1|1|cell-lborder|1ln>|<cwith|1|-1|3|3|cell-rborder|1ln>|<cwith|1|1|1|-1|cell-tborder|1ln>|<cwith|1|1|1|-1|cell-bborder|1ln>|<cwith|1|1|1|-1|cell-lborder|1ln>|<cwith|1|1|1|-1|cell-rborder|1ln>|<cwith|2|2|1|-1|cell-tborder|1ln>|<table|<row|<cell|<math|M<rsub|1>>>|<cell|Message
-  Type>|<cell|Description>>|<row|<cell|0>|<cell|Status>|<cell|<reference|sect-msg-status>>>|<row|<cell|1>|<cell|Block
-  Request>|<cell|<reference|sect-msg-block-request>>>|<row|<cell|2>|<cell|Block
-  Response>|<cell|<reference|sect-msg-block-response>>>|<row|<cell|3>|<cell|Block
-  Announce>|<cell|<reference|sect-msg-block-announce>>>|<row|<cell|4>|<cell|Transactions>|<reference|sect-msg-transactions>>|<row|<cell|5>|<cell|Consensus>|<cell|<reference|sect-msg-consensus>>>|<row|<cell|6>|<cell|Remote
-  Call Request>|<cell|>>|<row|<cell|7>|<cell|Remote Call
-  Response>|<cell|>>|<row|<cell|8>|<cell|Remote Read
-  Request>|<cell|>>|<row|<cell|9>|<cell|Remote Read
-  Response>|<cell|>>|<row|<cell|10>|<cell|Remote Header
-  Request>|<cell|>>|<row|<cell|11>|<cell|Remote Header
-  Response>|<cell|>>|<row|<cell|12>|<cell|Remote Changes
-  Request>|<cell|>>|<row|<cell|13>|<cell|Remote Changes
-  Response>|<cell|>>|<row|<cell|14>|<cell|FinalityProofRequest>|<cell|>>|<row|<cell|15>|<cell|FinalityProofResponse>|<cell|>>|<row|<cell|255>|<cell|Chain
-  Specific>|<cell|>>>>>|<label|tabl-message-types>List of possible network
-  message types.>
-
```

Comment:
So now we basically have only block announce request, response and transaction. are all the other messages were removed from Polkadot/substrate?

Reply:
Yes, now we just have block announce, block request, block response, grandpa votes, transactions and light client requests (see  #376).